### ISSUE TITLE
FailedValidation creating schedule with CRON_TZ

### ISSUE THREAD

**Issue by vmaglietta** [vmware-tanzu/velero#8224]
I am trying to create a schedue with Tome Zone.
After giving the command:
`velero schedule create example-schedule --schedule="CRON_TZ=America/New_York 0 3 * * *"`
I  am getting the following Validation Error:
```
Name:         example-schedule
Namespace:    velero
Labels:       <none>
Annotations:  <none>
API Version:  velero.io/v1
Kind:         Schedule
Metadata:
  Creation Timestamp:  2024-09-18T07:45:34Z
  Generation:          2
  Resource Version:    7839194
  UID:                 f3068145-1d4b-41cb-89be-86e804dd0bcb
Spec:
  Schedule:          CRON_TZ=America/New_York 0 3 * * *
  Skip Immediately:  false
  Template:
    Csi Snapshot Timeout:  0s
    Hooks:
    Included Namespaces:
      *
    Item Operation Timeout:  0s
    Metadata:
    Ttl:                           0s
  Use Owner References In Backup:  false
Status:
  Phase:  FailedValidation
  Validation Errors:
    invalid schedule: Expected exactly 5 fields, found 6: CRON_TZ=America/New_York 0 3 * * *
Events:  <none>

```
What am I missing?
`Image:         velero/velero:v1.14.1` 

Regards

**Comment by blackpiglet** [comment]
Your scenario-related PR is #7793.
This PR was merged into the main branch only.
The main branch is going to be released as v1.15.0.
By now, you need to use the main branch image to get this function.

**Comment by vmaglietta** [comment]
@blackpiglet thank you so much for your info. Sorry I missed #7793
